numpyro/distributions/continuous.py

```diff
@@ -71,7 +71,7 @@
     validate_sample,
     vec_to_tril_matrix,
 )
-from numpyro.util import is_prng_key
+from numpyro.util import add_diag, is_prng_key
```

Comment:
Super cool!!

Could you move this helper to `numpyro.distributions.util` instead?